assets/prefabs/redFlag.prefab

```diff
@@ -1,6 +1,8 @@
 {
     "parent" : "flag",
-    "RedFlag" : {},
+    "Flag" : {
+        "team" : "red"
+    },
     "LASTeam" : {
         "team" : "red"
     }
```

Comment:
I'm wondering if we still need this. With the flag component containing the team, this seems like duplicate information.